bundles/org.openhab.binding.somfytahoma/README.md

```diff
@@ -124,11 +128,19 @@ When a roller shutter-like thing receives STOP command, there are two possible b
 If you want to set the MY position of a roller shutter and you don't care the possible movement, try sending the MOVE command (OH2 does not know MY, so it stands for "move to MY position")
 
 ```
-sendCommand(ROLLERSHUTTER_THING, MOVE)
+sendCommand(CONTROL_CHANNEL, MOVE)
```

Comment:
Can you remove this examples? We should prefer the second one when sending commands to items.

Reply:
Removed